src/api/ParticipantManager.Experience.API/Functions/ScreeningEligibilityFunction.cs

```diff
@@ -44,13 +45,25 @@ public async Task<IActionResult> GetParticipantEligibility(
       var pathwayEnrolments = await crudApiClient.GetPathwayEnrolmentsAsync(participantId);
       if (pathwayEnrolments == null)
       {
-        logger.LogError("Failed to find pathway enrolments for NhsNumber: {@ParticipantId}",
+        logger.LogError("Failed to find pathway enrolments for Participant: {@ParticipantId}",
           new { ParticipantId = participantId });
         return new NotFoundObjectResult("Unable to find pathway enrolments");
       }
 
+      if (pathwayEnrolments.Count == 0)
```

Comment:
I think this entire if clause can be removed.

The shape of the response body returned in the OkObjectResult differs from the one returned at the end of the function, so it would confuse the consuming front-end when deserialising.

The Any clause in the access check on Line 66 will still work if the list of pathway enrolments is empty.